chart/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
Nit: We can remove this file (LICENSE), as the [LICENSE is in the repository](https://github.com/in-toto/archivista/blob/main/LICENSE).